server_admin/topics/events/admin.adoc

```diff
@@ -35,3 +35,9 @@ You can now view admin events.
 .Admin events
 image:images/admin-events.png[Admin events]
 
+When `Include Representation` switch is ON, it can lead to storing a lot of information in database. You can set a maximum length of the representation by using the `--spi-events-store-jpa-max-field-length` argument. This setting is useful if you want to adhere to the underlying storage limitation. For example:
```

Comment:
```suggestion
When the `Include Representation` switch is ON, it can lead to storing a lot of information in the database. You can set a maximum length of the representation by using the `--spi-events-store-jpa-max-field-length` argument. This setting is useful if you want to adhere to the underlying storage limitation. For example:
```